rocky/katalogus/views/plugin_detail.py

```diff
@@ -2,6 +2,7 @@
 from typing import Any
 
 from account.mixins import OrganizationView
+from dateutil import parser
```

Comment:
regression, dateutil is only present in dev. 
This was fixed in https://github.com/minvws/nl-kat-coordination/pull/3590